### PR TITLE
Update console.properties

### PR DESCRIPTION
**Pull request by bendeg**
Minor changes.

### CODE REVIEW

File: console/console.properties [hunk 1]
```diff
@@ -233,7 +234,7 @@
 #subject.new.account.notification=[${instanceName}] New account created
 
 # Encoding of the email templates
-# This "�" char should display nicely in a ISO 8859-1 configured editor
+# This "é" char should display nicely in a ISO 8859-1 configured editor
```
Comment (fvanderbiest) [inline console/console.properties]: This change probably broke file encoding.

It should be edited in a latin-1 configured editor. 
Github editor converts it to UTF-8, as stated in the notice.

Comment (fvanderbiest) [reply console/console.properties]: My bad, we do not mention github editor breaks file encoding.

**Comment by bendeg** [comment]
I'm not sure how to do that properly.

Should I just get rid of this PR, fork and make the changes locally ?

**Comment by fvanderbiest** [comment]
> Should I just get rid of this PR, fork and make the changes locally ?

Yes, there is no other way.
I'm sorry about that.

**Pull request closed without merging**